src/mozanalysis/frequentist_stats/bootstrap.py

```diff
@@ -284,7 +284,7 @@ def get_quantile_bootstrap_samples(
     sample_size = data.shape[0]
     samples = {
         f"{quantile:.1}": data[
-            np.random.binomial(sample_size + 1, quantile, num_samples)
+            np.random.binomial(sample_size - 1, quantile, num_samples)
```

Comment:
Does this translation from R imply that we've overcorrected here? Should it be simply `sample_size` instead of `sample_size - 1` to adjust for the 0 vs 1 indexing?

Reply:
When I tested `np.array([3.0, 3.0, 3.0]), quantiles_of_interest=[0.5], num_samples=10` with just `sample_size` I still got the indexing errors